<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelsDialog</class>
 <widget class="QDialog" name="LabelsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LabelsDialog
{
	background:white;
	border:2px solid #bbbbbb;
	border-radius:10px;
}

.QLineEdit
{
	border:2px solid #bbbbbb;
	border-radius: 5px;
}</string>
  </property>
  <widget class="QLineEdit" name="txtXLeft">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtXRight">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>85</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtYUp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtYDown">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLabelsMod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnLabelsMod
{
	border: 2px solid rgb(107,90,82);
	border-radius:10px;
	background: transparent;
	color: rgb(107,90,82);
}

#btnLabelsMod:hover
{
	background: rgb(107,90,82);
	color: white;
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtYUp</tabstop>
  <tabstop>txtYDown</tabstop>
  <tabstop>txtXRight</tabstop>
  <tabstop>txtXLeft</tabstop>
  <tabstop>btnLabelsMod</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnLabelsMod</sender>
   <signal>clicked()</signal>
   <receiver>LabelsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
